.circleci/config.yml

```diff
@@ -8,16 +8,40 @@ jobs:
       TRIGGER_BUILD_BRANCH: master
     steps:
     - run:
-        name: Install tools
+        name: Install tools and ipfs
         command: |
-          sudo apt-get update
-          sudo apt-get -y install libudev-dev libusb-1.0-0-dev
+            cd ..
+            sudo apt-get update
+            sudo apt-get -y install libudev-dev libusb-1.0-0-dev curl
+            # install IPFS
+            wget https://dist.ipfs.io/go-ipfs/v0.4.17/go-ipfs_v0.4.17_linux-amd64.tar.gz
+            tar xvfz go-ipfs_*.tar.gz
+            sudo cp ./go-ipfs/ipfs /usr/local/bin
+    - run:
+        name: Install platform-contracts from master
```

Comment:
Are platform-contracts used for tests only? In this case it is better to put this section as a separate run task before test running.